test/unit/services/policies/policies_list_service_test.rb

```diff
@@ -4,9 +4,15 @@
 
 class Policies::PoliciesListServiceTest < ActiveSupport::TestCase
 
+  APICAST_REGISTRY_URL = 'https://apicast-staging.proda.example.com/policies'
+
   def setup
     @service = Policies::PoliciesListService
-    ThreeScale.config.sandbox_proxy.stubs(apicast_registry_url: 'https://apicast-staging.proda.example.com/policies')
+    ThreeScale.config.sandbox_proxy.stubs(apicast_registry_url: APICAST_REGISTRY_URL)
+  end
+
+  def stub_apicast_request(status: 200, response_body: GATEWAY_API_MANAGEMENT_RESPONSE.to_json, headers: { 'Content-Type' => 'application/json' })
+    stub_request(:get, APICAST_REGISTRY_URL).to_return(status: status, response_body: response_body, headers: headers)
```

Comment:
sorry, my bad, this should be:
```
stub_request(:get, APICAST_REGISTRY_URL).to_return(status: status, body: response_body, headers: headers)
```

Reply:
🤦‍♂️

Reply:
hahahahaha but I pushed the fix, sorry 🙏 😂